omise-woocommerce.php

```diff
@@ -46,7 +54,7 @@ public function __construct() {
 	public function woocommerce_plugin_notice(){
 		?>
 		<div class="error">
-			<p><?php echo __( 'Plugin <strong>deactivated</strong>. The Omise WooCommerce plugin requires <strong>WooCommerce</strong> to be installed and active.', 'omise' ); ?></p>
+			<p><?php echo __( 'Omise WooCommerce plugin requires <strong>WooCommerce</strong> to be activated.', 'omise' ); ?></p>
```

Comment:
Shouldn't have removed "The" here

Reply:
@jonrandy fixed here: https://github.com/omise/omise-woocommerce/pull/94/commits/3eee88183aadb79be16d8bbda827611d88ded404